modules/hmmcopy/generatemap/main.nf

```diff
@@ -0,0 +1,38 @@
+process HMMCOPY_GENERATEMAP {
+    tag '$bam'
+    label 'process_long'
+
+    def VERSION = '0.1.1'
```

Comment:
This should go outside the module itself. The example on the [guidelines](https://nf-co.re/developers/adding_modules#general) shows this: https://github.com/nf-core/modules/blob/master/modules/homer/annotatepeaks/main.nf

Reply:
Moved VERSION outside. Tried to allow compressed FASTA but the perl script looks for the name of the fasta file and makes indexes accordingly.